lib/cisco_node_utils/bgp_af.rb

```diff
@@ -424,15 +424,14 @@ def networks=(should_list)
       [:add, :remove].each do |action|
         Cisco::Logger.debug("networks delta #{@get_args}\n #{action}: " \
                             "#{delta_hash[action]}")
-        delta_hash[action].each do |network, route_map_policy|
+        delta_hash[action].each do |network, route_map|
           state = (action == :add) ? '' : 'no'
           network = Utils.process_network_mask(network)
-          unless route_map_policy.nil?
-            route_map = "route-map #{route_map_policy}"
-            route_policy = "route-policy #{route_map_policy}"
+          unless route_map.nil?
+            route_map = "route-map #{route_map}"    if platform == :nexus
+            route_map = "route-policy #{route_map}" if platform == :ios_xr
```

Comment:
A nit, but I would change line 431 to:

route_map = "route-map #{route_map}"    unless platform == :ios_xr

If another platform is added, at least it will execute one of them.  If you feel that "route-policy" is more likely for other platforms, then make that one the default.